Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/ToolBar/EditorToolBar.cpp

```diff
@@ -35,6 +40,23 @@ namespace AzToolsFramework
         m_toolBarItems.insert({ sortKey, ToolBarItem(ToolBarItemType::Action, AZStd::move(actionIdentifier)) });
     }
 
+    void EditorToolBar::AddActionWithSubMenu(int sortKey, AZStd::string actionIdentifier, AZStd::string subMenuIdentifier)
```

Comment:
I was going to suggest you use a std move when you pass the string into GetAction/GetMenu maybe it would be better in this case to pass by const reference?

Reply:
Thank for pointing it out, I missed it :D 
I will try to use moves for consistency with the rest of the API if possible.

Reply:
Added a move for `actionIdentifier`, but since we don't need to reuse the `subMenuIdentifier` I've changed that to const ref. Thanks!